controller/webhook.go

```diff
@@ -20,32 +20,35 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"log"
```

Comment:
Are all these new import needed?  For example I don't see `log` being used.

Reply:
My changes did not need any new packages to be imported.
log was already in use.
Looks like my Visual Code editor did its own "smart" editing and grouped these packages differently, making it appear like there were new packages added.